awscli/customizations/cloudformation/artifact_exporter.py

```diff
@@ -47,6 +47,13 @@ def is_s3_url(url):
         return False
 
 
+def is_https_s3_url(url):
+    if isinstance(url, six.string_types) \
+        and url.startswith("https://s3.amazonaws.com/"):
```

Comment:
After testing, it looks like CloudFormation also supports urls with the `http` protocol as well. I think we need to support everything that's listed on [this page](http://docs.aws.amazon.com/AmazonS3/latest/dev/VirtualHosting.html#VirtualHostingExamples).

Using `re.match("https?://[\w.\d\-]*s3[\d\w-]*\.amazonaws\.com/", url)` would theoretically match all on that page (unless I missed some) except CNAME urls. To support those we should just check for the `http` protocol and keep moving.